src/ActivityForm.css

```diff
@@ -0,0 +1,47 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+h1 {
+  background: #8b0000;
+  color: whitesmoke;
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  width: 100%;
+  margin: 0;
+}
+
+div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: Helvetica;
+}
```

Comment:
**📢 Font Family Cascade**
_It's often more efficient to set the font family for the entire website or a major section in a single, higher-level place like the 'body' or a parent container. This ensures consistency and reduces redundancy._